alphagov/paas-product-pages
# Update dependencies and use new link hover styles

## What

- Regular dependency update
- Use [new link hover styles introduced in govuk-frontend v3.12.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.12.0)

## Visual change (bolder hover state)

![Screenshot 2021-05-14 at 09 08 08](https://user-images.githubusercontent.com/3758555/118241542-2e4eae80-b494-11eb-8c5f-66d3e0d0f709.png)

## How to test
- install new dependencies (`npm install`)
- build the site (`npm run build`)
- `cd` into `out` folder and serve the page (`npx serve`)
- check page on localhost:5000 for any issues
